diff --git a/node_modules/@angular-devkit/build-angular/src/angular-cli-files/models/webpack-configs/browser.js b/node_modules/@angular-devkit/build-angular/src/angular-cli-files/models/webpack-configs/browser.js
index 734bbb1..a6064f7 100644
--- a/node_modules/@angular-devkit/build-angular/src/angular-cli-files/models/webpack-configs/browser.js
+++ b/node_modules/@angular-devkit/build-angular/src/angular-cli-files/models/webpack-configs/browser.js
@@ -83,7 +83,16 @@ function getBrowserConfig(wco) {
             }),
             ...extraPlugins,
         ],
-        node: false,
+        node: {
+          module: 'empty',
+          dgram: 'empty',
+          dns: 'mock',
+          fs: 'empty',
+          http2: 'empty',
+          net: 'empty',
+          tls: 'empty',
+          child_process: 'empty',
+        },
     };
 }
 exports.getBrowserConfig = getBrowserConfig;
